global_helpers/panther_oss_helpers.py

```diff
@@ -16,6 +16,31 @@ class BadLookup(Exception):
 class PantherBadInput(Exception):
     """Error returned when a Panther helper function is provided bad input."""
 
+def build_client(resource):
+    """Function builds resource client that assumes Panther audit role"""
+    tmp = resource['ResourceId'].split(':')
+    resource_type = tmp[2]
+    region = tmp[3]
+    account = tmp[4]
+    role_arn = f'arn:aws:iam::{account}:role/PantherAuditRole-{region}'
+    sts_connection = boto3.client('sts')
+    acct_b = sts_connection.assume_role(
+        RoleArn=role_arn,
+        RoleSessionName="lambda_assume_audit_role"
+    )
```

Comment:
Let's include the changes we discussed here, having the caller specify a service etc.

Reply:
Testing then sending the code changes we made.